parquet/src/main/java/org/apache/iceberg/parquet/TypeWithSchemaVisitor.java

```diff
@@ -37,7 +37,7 @@
  */
 public class TypeWithSchemaVisitor<T> {
   @SuppressWarnings({"checkstyle:VisibilityModifier", "checkstyle:IllegalType"})
```

Comment:
Is the checkstyle suppression still valid?

Reply:
at least the `checkstyle:VisibilityModifier` is still valid: `TypeWithSchemaVisitor.java:39:32: Variable 'fieldNames' must be private and have accessor methods. [VisibilityModifier]`

Reply:
Yeah, I thought the visibility modifier one would be. But the illegal type seems like it would be no longer needed.